consensus/merge/src/main/java/org/hyperledger/besu/consensus/merge/blockcreation/PayloadIdentifier.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright contributors to Hyperledger Besu
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+ * the License. You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+ * an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ * specific language governing permissions and limitations under the License.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+package org.hyperledger.besu.consensus.merge.blockcreation;
+
+import org.hyperledger.besu.plugin.data.Quantity;
+
+import java.math.BigInteger;
+import java.util.Random;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonValue;
+import org.apache.tuweni.units.bigints.UInt64;
+
+public class PayloadIdentifier implements Quantity {
+
+  private final UInt64 val;
+  static final Random rand = new Random();
+
+  @JsonCreator
+  public PayloadIdentifier(final String payloadId) {
+    this(Long.decode(payloadId));
+  }
+
+  public PayloadIdentifier(final Long payloadId) {
+    this.val = UInt64.valueOf(payloadId);
+  }
+
+  public static PayloadIdentifier random() {
```

Comment:
Seems like this is only used for testing, and could possibly contribute to flakey tests. Any reason not to choose static values for testing instead of generating newly random ones each time?

Reply:
The payload identifier is used by the consensus client to fetch block payloads.  Payloads are not generated in this PR however, that is why it looks like a test concern.  I don't think there is any risk in having a deterministic payload identifier, so I think we can use a payload header hash instead and that should be sufficiently unique.